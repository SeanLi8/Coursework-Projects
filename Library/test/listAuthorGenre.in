add collection books.csv
add collection books2.csv
list genres
list authors
genre not a genre
genre yes
genre Romantic Comedy
genre Comedy
genre Historical Fiction
author sean li
author Homer
author Akasaka Aka
exit

