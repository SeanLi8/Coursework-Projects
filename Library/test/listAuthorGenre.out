user: 26 books successfully added.

user: 8 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
Romantic Comedy
Shounen
Sports
War

user: Akasaka Aka
Bob Kane
Douglas Adams
Fujiwara Hiro
Furudate Haruichi
Harper Lee
Homer
Inoue Takehik0
Isayama Hajime
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
Obata Takeshi
Saeki Shun
T.S. Eliot
Togashi Yoshihiro
Virgil
W.E. Johns

user: No books with genre not a genre.

user: No books with genre yes.

user: Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)
Kaichou wa Maid-sama! (Fujiwara Hiro)

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: No books by sean li.

user: The Iliad (Homer)
The Odyssey (Homer)

user: Kaguya-sama wa Kokurasetai: Tensai-tachi no Renai Zunousen (Akasaka Aka)

user: Ending Library process.
